applications/adv_networking_bench/cpp/CMakeLists.txt

```diff
@@ -107,3 +101,39 @@ install(
         ../adv_networking_bench_rmax_rx_yaml
   DESTINATION bin/adv_networking_bench/cpp
 )
+
+# Add testing
+if(BUILD_TESTING)
+  # Get the BUS ID of the interface doing Rx in the config
```

Comment:
Updated commands look good, is this for loopback only? Would it be reasonable to add a short note explaining the expected test hardware configuration here or in the README, or too complicated to include in HoloHub?

Reply:
no that's fair. yes the current config sort of defaults to the IGX loopback currently, though we could also make that generic and have the test override it and call the test `dpdk_loopback` or something

Reply:
I think my suggestion above could be for a future MR, I'll add comments for now as you suggested.

Reply:
done here: https://github.com/nvidia-holoscan/holohub/compare/e12d8e5fd48c24bb2d4b6e8fd68481ae261f8b2e..1999ee7f4087ca448b971473c118a6c5132f073c#diff-43cc86cb9a282f7e8af6006f07458870a547872712097c5cef73407e574e5e71R108